pkg/apis/cloudcore/v1alpha1/validation/validation.go

```diff
@@ -83,12 +82,6 @@ func ValidateModuleCloudHub(c cloudconfig.CloudHub) field.ErrorList {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("address"),
 			c.UnixSocket.Address, "unixSocketAddress must has prefix unix://"))
 	}
-	s := strings.SplitN(c.UnixSocket.Address, "://", 2)
-	if len(s) > 1 && !utilvalidation.FileIsExist(path.Dir(s[1])) {
```

Comment:
Thanks for the fix!
Could we create the `path.Dir(s[1])` if it doesn't exist? Ref: https://github.com/kubeedge/kubeedge/blob/master/pkg/apis/edgecore/v1alpha1/validation/validation.go#L50 
@zhuguihua